client/src/App.js

```diff
@@ -1,12 +1,99 @@
-import React from 'react';
+import React, { Component }from 'react';
 import './App.css';
+import { Router } from '@reach/router'
+import { Link } from '@reach/router'
+import NavBar from './Components/NavBar'
+import Home from './Components/Home'
+import About from './Components/About'
+import Projects from './Components/Projects'
+import Hobbies from './Components/Hobbies'
+import Work from './Components/Work'
+import Contact from './Components/Contact'
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Hello portfolio!</h1>
-    </div>
-  );
+
+// this changes the color of the navigation bar based on which page the user is on
```

Comment:
This note would be more relevant living down inside of your render, closer to where you're defining the colors